app/controllers/api/base_controller.rb

```diff
@@ -4,9 +4,8 @@ module Api
   #
   Initializer.new.go
 
-  class BaseController < ApplicationController
```

Comment:
Not tested possible, but you won't know until it's exercised. What if it's exercised not by us but via an edge case via customer scripts, we won't know until it's out in the field.  

I tend to prefer a safer route, i.e. my comfort level would be to do this very early in the dev release, i.e. early F-release.